signs-work-order-map/app/page.tsx

```diff
@@ -2,21 +2,19 @@
 import styles from "./page.module.css";
 import Map from "@/components/Map";
 import { useIFrameMessenger } from "@/utils/iFrameMessenger";
-import { useFormatSignsRecords, useFormatLocation } from "@/utils/mapUtils";
+import { useFormatSignsRecords } from "@/utils/mapUtils";
 
 export default function Home() {
   const knackPayload = useIFrameMessenger();
 
-  console.log(knackPayload);
-
-  const location = useFormatLocation(knackPayload);
+  // const location = useFormatLocation(knackPayload);
```

Comment:
this will come back into use when I tackle https://github.com/cityofaustin/atd-data-tech/issues/23268 next